korgw/src/commonMain/kotlin/com/soywiz/korag/AGObjects.kt

```diff
@@ -80,6 +80,8 @@ class AGTexture(
     private val logger = Logger("AGTexture")
     var isFbo: Boolean = false
     var requestMipmaps: Boolean = false
+    var baseMipmapLevel: Int? = null
+    var maxMipmapLevel: Int? = null
```

Comment:
Default value is null because if you don't specify level, GL generates some value on its own